diff --git a/dist/publish.js b/dist/publish.js
index 9ff19090801bbd6791f6ad29a0ae27bc77bd14fd..50c042c08abd010ed580d76e25cbe4af493fe56b 100644
--- a/dist/publish.js
+++ b/dist/publish.js
@@ -53,8 +53,9 @@ export async function publish(packages, options) {
             log.info(`Creating git tag.`);
             execSync(`git tag ${gitTag}`);
             execSync("git push --tags");
-            log.info(`Creating GitHub release notes.`);
-            execSync(`gh release create ${gitTag} --notes '${changelog}'`);
+            const prerelease = name.startsWith("@auth") ? "--prerelease " : ""
+            console.log(`Creating GitHub release notes${prerelease ? " (as prerelease)" : ""}`);
+            execSync(`gh release create ${gitTag} ${prerelease}--notes '${changelog}'`);
         }
     }
     if (dryRun) {
