diff --git a/dist/publish.js b/dist/publish.js
index 1d39a76a5625a10637e3cf23f3a652441702cfb2..4b69b30645db1d1e84c8b6987a23b53c06f66438 100644
--- a/dist/publish.js
+++ b/dist/publish.js
@@ -54,8 +54,9 @@ export async function publish(packages, options) {
             console.log(`Creating git tag...`);
             execSync(`git tag ${gitTag}`);
             execSync("git push --tags");
-            console.log(`Creating GitHub release notes...`);
-            execSync(`gh release create ${gitTag} --notes '${changelog}'`);
+            const prerelease = name.startsWith("@auth") ? "--prerelease " : ""
+            console.log(`Creating GitHub release notes${prerelease ? " (as prerelease)" : ""}...`);
+            execSync(`gh release create ${gitTag} ${prerelease}--notes '${changelog}'`);
         }
     }
     console.log("Pushing commits");