diff --git a/dist/analyze.js b/dist/analyze.js
index ea0568191c9dfbee419e8bb322cc57c14d7b2784..a8bee1069ab1fe629515cdd4c83aa1e419779a8c 100644
--- a/dist/analyze.js
+++ b/dist/analyze.js
@@ -137,7 +137,8 @@ export async function analyze(config) {
                 dependents: [],
             });
     }
-    const result = Array.from(packagesToRelease.values());
+    // TODO: Re-enable when next-auth v5 is stable
+    const result = Array.from(packagesToRelease.values()).filter(p => p.name !== "next-auth")
     if (config.peek) {
         log.peekInfo("Following packages can be released:\n", ...result.map((p) => `  - ${bold(p.name)}: ${p.oldVersion} -> ${p.newVersion}`));
         exit();
diff --git a/dist/publish.js b/dist/publish.js
index 09a0c10584159beefa7a915ad92d46601da5ad45..bbb63c7e687c1ca4b715b3a80a512d973a729da7 100644
--- a/dist/publish.js
+++ b/dist/publish.js
@@ -22,6 +22,9 @@ export async function publish(packages, options) {
         await pkgJson.update(pkg.relativeDir, { version: pkg.newVersion });
         log.info("package.json file has been written, publishing.");
         let npmPublish = `pnpm publish --access public --registry=https://registry.npmjs.org --no-git-checks`;
+        if(pkg.name === "next-auth") {
+            npmPublish += " --tag beta";
+        }
         if (dryRun) {
             npmPublish += " --dry-run";
             if (!options.verbose)
@@ -55,7 +58,9 @@ export async function publish(packages, options) {
             execSync(`git tag ${gitTag}`);
             execSync("git push --tags");
             log.info(`Creating GitHub release notes.`);
-            execSync(`gh release create ${gitTag} --notes '${changelog}'`);
+            const prerelease = name.startsWith("@auth") ? "--prerelease " : ""
+            console.log(`Creating GitHub release notes${prerelease ? " (as prerelease)" : ""}`);
+            execSync(`gh release create ${gitTag} ${prerelease}--notes '${changelog}'`);
         }
     }
     if (dryRun) {
