diff --git a/dist/theme.js b/dist/theme.js
index 1483a4b4ec69583aa3086eac83b2b31ae8bb6777..6b90b17261e14b8a49e52d0885038dbff4b6b503 100644
--- a/dist/theme.js
+++ b/dist/theme.js
@@ -209,52 +209,6 @@ class MarkdownTheme extends typedoc_1.Theme {
                 directory: 'modules',
                 template: this.getReflectionTemplate(),
             },
-            {
-                kind: [typedoc_1.ReflectionKind.Namespace],
-                isLeaf: false,
-                directory: 'modules',
-                template: this.getReflectionTemplate(),
-            },
-            {
-                kind: [typedoc_1.ReflectionKind.Enum],
-                isLeaf: false,
-                directory: 'enums',
-                template: this.getReflectionTemplate(),
-            },
-            {
-                kind: [typedoc_1.ReflectionKind.Class],
-                isLeaf: false,
-                directory: 'classes',
-                template: this.getReflectionTemplate(),
-            },
-            {
-                kind: [typedoc_1.ReflectionKind.Interface],
-                isLeaf: false,
-                directory: 'interfaces',
-                template: this.getReflectionTemplate(),
-            },
-            ...(this.allReflectionsHaveOwnDocument
-                ? [
-                    {
-                        kind: [typedoc_1.ReflectionKind.TypeAlias],
-                        isLeaf: true,
-                        directory: 'types',
-                        template: this.getReflectionMemberTemplate(),
-                    },
-                    {
-                        kind: [typedoc_1.ReflectionKind.Variable],
-                        isLeaf: true,
-                        directory: 'variables',
-                        template: this.getReflectionMemberTemplate(),
-                    },
-                    {
-                        kind: [typedoc_1.ReflectionKind.Function],
-                        isLeaf: true,
-                        directory: 'functions',
-                        template: this.getReflectionMemberTemplate(),
-                    },
-                ]
-                : []),
         ];
     }
     onBeginRenderer(event) {