diff --git a/dist/partials/members.js b/dist/partials/members.js
index 4fe981318e71ea716da9fca652ce6a0dc5a3e6df..e888cb3a4bc0a838229ae00b409b748a236b254c 100644
--- a/dist/partials/members.js
+++ b/dist/partials/members.js
@@ -3,7 +3,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.members = void 0;
 const typedoc_1 = require("typedoc");
 const els_1 = require("../support/els");
-const helpers_1 = require("../support/helpers");
+// const helpers_1 = require("../support/helpers");
 function members(context, container) {
     var _a;
     const md = [];
@@ -18,14 +18,14 @@ function members(context, container) {
     }
     else {
         (_a = container.groups) === null || _a === void 0 ? void 0 : _a.filter((group) => !group.allChildrenHaveOwnDocument()).forEach((group) => {
-            const headingLevel = (0, helpers_1.getGroupHeadingLevel)(container);
+            // const headingLevel = (0, helpers_1.getGroupHeadingLevel)(container);
             if (group.categories) {
                 group.categories.forEach((groupItem) => groupItem.children.forEach((item) => {
                     md.push(context.partials.member(item));
                 }));
             }
             else {
-                md.push((0, els_1.heading)(headingLevel, group.title));
+                // md.push((0, els_1.heading)(headingLevel, group.title));
                 group.children
                     .filter((item) => !item.hasOwnDocument)
                     .forEach((groupChild, index) => {