diff --git a/dist/theme.js b/dist/theme.js
index 1483a4b4ec69583aa3086eac83b2b31ae8bb6777..c30e7a4f7785fc230099e8b904040dd4aa57c38e 100644
--- a/dist/theme.js
+++ b/dist/theme.js
@@ -221,12 +221,12 @@ class MarkdownTheme extends typedoc_1.Theme {
                 directory: 'enums',
                 template: this.getReflectionTemplate(),
             },
-            {
-                kind: [typedoc_1.ReflectionKind.Class],
-                isLeaf: false,
-                directory: 'classes',
-                template: this.getReflectionTemplate(),
-            },
+            // {
+            //     kind: [typedoc_1.ReflectionKind.Class],
+            //     isLeaf: false,
+            //     directory: 'classes',
+            //     template: this.getReflectionTemplate(),
+            // },
             {
                 kind: [typedoc_1.ReflectionKind.Interface],
                 isLeaf: false,